dev/protos/firestore/bundle.proto

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// This file defines the format of Firestore bundle file/stream. It is not a part of the
+// Firestore API, only a specification used by Server and Client SDK to write and read
+// bundles.
+
+syntax = "proto3";
+
+package firestore.proto;
```

Comment:
This is now just `firestore` if my understanding is correct.

Reply:
Done.